config.go

```diff
@@ -459,6 +459,18 @@ type Config struct {
 	// Optional: defaults to 4
 	DataIterationConcurrency int
 
+	// This specify the number of concurrent goroutines, each writing an
+	// equally sized chunk of data to the target.
+	//
+	// Number of records per goroutine is calculated using
+	// `DataIterationBatchSize / RowBatchConcurrency`.
```

Comment:
I've clarified the rounding behaviour. Let's wait with replacing `DataIterationBatchSize`, I think it would be beneficial for the user to set batch size and concurrency, but before we have tested the effect of chunking I would like this PR to have as small footprint as possible for now.